curves/CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
```

Comment:
Should we move this to a top level file, and remove the references to `curves`?

Reply:
done